### DISCUSSION TITLE
nextbox_ui_plugin migration

### DISCUSSION THREAD

**Discussion by aledr** [netbox-community/netbox#9143] [Q&A] [answered]
Hi.

I saw this message while running the latest update (3.2.1):

`
Running migrations:
  No migrations to apply.
  Your models in app(s): 'nextbox_ui_plugin' have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
`

But `python manage.py makemigrations` throws:

`
CommandError: This command is available for development purposes only. It will
NOT resolve any issues with missing or unapplied migrations. For assistance,
please post to the NetBox discussion forum on GitHub:
    https://github.com/netbox-community/netbox/discussions
`

Can you help?

**Comment by marcucci-DAE** [comment]
Hi all, 
I have the same problem....

Somebody can help you?

**Comment by DanSheps** [comment, marked as answer]
You need to contact the maintainer of the plugin and have them fix this.  This isn't a end-user fix unfortunately.

Alternatively, you can disable the plugin for now.
